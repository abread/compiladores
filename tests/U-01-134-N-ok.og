auto f() { return 5, 61.0, "string"; }
public int og() {
  auto a = f();
  write a@1;
  write a@2;
  write a@3;
  return 0;
}

