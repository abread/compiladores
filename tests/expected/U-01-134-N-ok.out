56.1E1string
